solidity/contracts/AltheaDB.sol

```diff
@@ -299,6 +269,11 @@ contract AltheaDB {
         if (is_user_admin(msg.sender)) {
             uint256 index = get_index_of_id(entry, state_registeredUsers);
             delete_array_entry(index, state_registeredUsers);
+
+            state_registeredAddr[entry.eth_addr] = false;
+            state_registeredIps[entry.mesh_ip] = false;
+            state_registeredKey[entry.wg_key] = false;
```

Comment:
Ok so there are 3 states now, true, false, and null? So when we try to look up a user who's not in this list is successfully resolves as false? Clever strategy. Couldn't we free the storage here when we delete the user?